src/org/jitsi/impl/neomedia/RTPConnectorInputStream.java

```diff
@@ -751,6 +745,17 @@ private void runInReceiveThread()
             {
                 receive(p);
             }
+            catch (SocketTimeoutException ste)
```

Comment:
I'm confused because in the comment it says that we need to handle these exceptions but we no longer give the calling code an opportunity to handle these exceptions (because of the continue, there is no re-throw).

Reply:
Thanks for the offline discussion Boris. Continuing actually handles the exception.